sp-privacy/LICENSE

```diff
@@ -0,0 +1,845 @@
+sp-privacy
+Copyright © 2021 System1 and other contributors
```

Comment:
this might need to be Surfboard Holdings BV instead?

Reply:
ah good point, i'll update